db-developments/sql/_init.sql

```diff
@@ -94,12 +94,12 @@ CORRECTIONS:
 	date_statusr
 	date_statusx
 */
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'stories_prop');
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'bin');
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'bbl');
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'date_lastupdt');
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'date_filed');
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'date_statusd');
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'date_statusp');
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'date_statusr');
-CALL apply_correction('_INIT_devdb', '_manual_corrections', 'date_statusx');
\ No newline at end of file
+CALL apply_correction('{{BUILD_SCHEMA}}', '_INIT_devdb', '_manual_corrections', 'stories_prop');
```

Comment:
I believe args need to be in format `:'BUILD_SCHEMA'`. I think this is just passing that string literal at the moment, hence why no schema is being used and therefore no corrections are being applied

Reply:
Ah yeah they're not being applied either. Only lat/long corrections which have a different mechanism

Reply:
yea I tried `:'BUILD_SCHEMA'` first and [the build failed](https://github.com/NYCPlanning/data-engineering/actions/runs/5693851614/job/15433833407)

Reply:
Looks like a mismatch of variable name? `BUILD_ENGINE_SCHEMA` in the `psql` call, `:'BUILD_SCHEMA;` in the sql file

Reply:
that's what I get for trying to shorten the variable name

Reply:
`corrections_applied.csv` is still 59 B. added a sql NOTICE to see what schema it's using and re-running